tutorials/singularity-containers-with-cloud-build/env.sh

```diff
@@ -0,0 +1,4 @@
+ORG=[YOUR_ORG]
+BILLING_ACCOUNT=[YOUR_BILLING_ACCOUNT_NAME]
+PROJECT=[NAME FOR THE PROJECT YOU WILL CREATE]
```

Comment:
any reason not to stick with the convention of using underscores between words for the last two?

Reply:
Nope. Done